### ISSUE TITLE
Font not working on VS Code for mac 

### ISSUE THREAD

**Issue by sohilkaushal** [rubjo/victor-mono#47]
Hey,

I just updated my Mac and the font stopped working all together.
As of now I have tried the following:
- reinstall the font
- reinstall VSCode
- reset my machine( I had to do it anyway because Catalina update broke some of my other utilities as well).

I don't know if this is something because of Homebrew.
So, it would be great if you could look into it.

Cherrs,

Sohil

**Comment by rubjo** [comment]
Hi Sohil!

That’s strange.
- Does Homebrew install the font and make it available in other apps?
- Do other fonts install correctly using Homebrew?
- Are VS Code preferences set to use the font?
- Do other fonts appear correctly in VS Code?

**Comment by sohilkaushal** [comment]
Hey @rubjo ,

So I just checked it out.

The font is now installing in VSCode it was my stupid config that was causing the errors. 
However, I still don't understand how can I enable italics as depicted in the website. 


Thanks in advance.

Sohil
<img width="1080" alt="Screen Shot 2019-10-12 at 11 45 54 am" src="https://user-images.githubusercontent.com/25654505/66691955-da8f4880-ece5-11e9-87e0-ee36fdd547df.png">

**Comment by rubjo** [comment]
You need to install and switch to a theme which specifies that parts of your code should be set in italics. I’m guessing that if you flick through the list of themes VS Code ships with, some of them have this?

**Comment by sohilkaushal** [comment]
Hey,

I have tried a few themes however the cursive's don't seem to be triggering for some reason.
Cheers
Sohil
<img width="1080" alt="Screen Shot 2019-10-12 at 8 25 32 pm" src="https://user-images.githubusercontent.com/25654505/66698958-72188980-ed2e-11e9-9613-650c5d04d4b9.png">

**Comment by rubjo** [comment]
I would check in Font Book if the font’s italic style is installed and not disabled for some reason.

**Comment by sohilkaushal** [comment]
> I would check in Font Book if the font’s italic style is installed and not disabled for some reason.
I have checked the font book and verified that the font is installed further, I have tested the font in webstorm and it is working

**Comment by rubjo** [comment]
Not sure what's wrong here, italics work fine in VS Code on Catalina here. I'm using VS Code Insiders, if that matters.

What does your VS Code font settings look like?
What does you Font Book entries for Victor Mono look like?

**Comment by sohilkaushal** [comment]
<img width="319" alt="Screen Shot 2019-10-14 at 6 54 59 pm" src="https://user-images.githubusercontent.com/25654505/66736189-3dced580-eeb4-11e9-873c-83e4f4b3b198.png">
Hey mate,

here are the screenshots of my font book and VSCode Settings

```json
{
  "extensions.autoCheckUpdates": true,
  "workbench.settings.editor": "json",
  "workbench.iconTheme": "material-icon-theme",
  "workbench.startupEditor": "none",
  "workbench.fontAliasing": "auto",
  "search.useIgnoreFiles": true,
  "zenMode.hideLineNumbers": false,
  "zenMode.hideActivityBar": false,
  "files.autoSave": "afterDelay",
  "files.autoSaveDelay": 1000,
  "files.simpleDialog.enable": true,
  "editor.smoothScrolling": true,
  "editor.formatOnPaste": true,
  "editor.wordWrap": "on",
  "editor.fontFamily": "Victor Mono",
  "editor.fontSize": 13,
  "editor.fontWeight": "500",
  "editor.fontLigatures": true,
  "editor.suggestSelection": "first",
  "editor.cursorSmoothCaretAnimation": true,
  "editor.cursorBlinking": "solid",
  "editor.renderIndentGuides": true,
  "editor.renderWhitespace": "boundary",
  "editor.copyWithSyntaxHighlighting": false,
  "editor.showFoldingControls": "mouseover",
  "editor.matchBrackets": false,
  "editor.tabSize": 2,
  "editor.rulers": [
    80,
  ],
  "breadcrumbs.enabled": true,
  "terminal.integrated.showExitAlert": false,
  "terminal.integrated.fontFamily": "Monoid HalfLoose",
  "terminal.integrated.fontSize": 11,
  "terminal.integrated.cursorBlinking": true,
  "terminal.integrated.cursorStyle": "line",
  "emmet.includeLanguages": {
    "javascript": "javascriptreact"
  },
  "vsintellicode.modify.editor.suggestSelection": "automaticallyOverrodeDefaultValue",
  "npm.enableScriptExplorer": true,
  "npm.autoDetect": "on",
  "explorer.confirmDelete": false,
  "window.zoomLevel": 0,
  "editor.detectIndentation": false,
  "search.useGlobalIgnoreFiles": true,
  "[javascript]": {
    "editor.defaultFormatter": "vscode.typescript-language-features"
  },
  "explorer.confirmDragAndDrop": false,
  "editor.minimap.enabled": false,
}
```

**Comment by rubjo** [comment]
K, seems that there is something preventing the italic from being applied in VS Code unless `editor.fontWeight` is set to `"600"`. Doesn't seem to happen on Windows. Will investigate this.

**Comment by sohilkaushal** [comment]
Cool that's good.

BTW I'll be happy to help

**Comment by rubjo** [comment]
The developer of the font editing software I use pointed out the obvious solution: uninstalling the seven oblique styles should make the italics work.

While testing and verifying that, I noticed another issue: Selecting weights 100-300 doesn't seem to make any difference in VS Code for me, even though they should correspond to the Thin and Extra Light weights. At least we have the italics sorted. 😄

**Comment by sohilkaushal** [comment]
That's cool.

Thanks for your help😃

**Comment by stdavis** [comment]
I had to manually uninstall all of the oblique styles also to get the cursive to work in VSCode on macOS (Catalina). Thanks for the tip, @rubjo. Seems like a hack but I'm happy that it's working.

**Comment by rubjo** [comment]
Yeah, hopefully they’ll get it sorted: https://github.com/microsoft/vscode/issues/82122

**Comment by patrick12james** [comment]
> The developer of the font editing software I use pointed out the obvious solution: uninstalling the seven oblique styles should make the italics work.
> 
> While testing and verifying that, I noticed another issue: Selecting weights 100-300 doesn't seem to make any difference in VS Code for me, even though they should correspond to the Thin and Extra Light weights. At least we have the italics sorted. 😄

How or where would you uninstall the 7 oblique styles?

**Comment by rubjo** [comment]
Deleting the oblique font files from your fonts folder. What’s the problem, I think everything should be working out of the box with VS Code now?

**Comment by patrick12james** [comment]
Yeah, at first I didn't know where they were located then found and deleted
them and it still didn't work so for some reason I had to copy this code:
 https://gist.github.com/mohokh67/63bd7d6fb1f2bb381e47cfb0478de071
 and then got it going.
Thanks for checking back

**Comment by PavanKumarCB** [comment]
> Hey,
> 
> I have tried a few themes however the cursive's don't seem to be triggering for some reason. Cheers Sohil <img alt="Screen Shot 2019-10-12 at 8 25 32 pm" width="1080" src="https://user-images.githubusercontent.com/25654505/66698958-72188980-ed2e-11e9-9613-650c5d04d4b9.png">

Heyy how do you enable the oblique text? I have selected Victor Mono oblique itself in my font book yet it doesnt seem to show up in VSCode

**Comment by kenmcd** [comment]
This issue should not have been closed.
The fonts available in the v1.5.5 release here are badly broken.
They are not going to work properly in many applications.

The easiest fix is to get the versions from Google Fonts.
Those fonts have been fixed and should work properly.
https://fonts.google.com/specimen/Victor+Mono
